quickwin/KAD-4376_gallery_arrow_dot_styles

gilbert-hernandez opened the pull request:
[https://stellarwp.atlassian.net/browse/KAD-4376](https://stellarwp.atlassian.net/browse/KAD-4376)

This adds arrow and dot color styles for gallery block carousels. This is a feature that requires the pro 2.7.7.

mark-c-woodard reviewed (changes requested):
https://www.loom.com/share/13185c1f62b14775bcd7410f5a7d871e?sid=2b918e31-48f1-452f-b57a-82bdbdf2ab40

gilbert-hernandez commented:
Moved the arrow settings to a new KadencePanelBody in the Style tab. Fixed the issue where the arrow opacity wasn't updated when the user hovered over the gallery.
